### ISSUE TITLE
508 accessibility issues when using filter

### ISSUE THREAD

**Issue by bhagadorn** [ag-grid/ag-grid-enterprise#207]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid-enterprise/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
ag-grid with Filter options does not pass 508.  The Filter fields do not get read by JAWS or NVDA 

**Expected behavior**
Expect the Filter fields to behave like ordinary input fields: JAWS should announce the name of each field as it gets focus.

**Minimal reproduction of the problem with instructions**
add Filter to any ag-grid. Run JAWS of NVDA.  Header cols are announced correctly but the Filter fields are not

**What is the motivation / use case for changing the behavior?**
Our application must be 508 compliant

**Please tell us about your environment:**
Windows with IE 11 and Chrome

* **ag-Grid-Enterprise version:** 2.0.X
Latest version
